Sources/FronteggSwift/models/plist/FronteggPlist.swift

```diff
@@ -65,8 +67,18 @@ struct FronteggPlist: Decodable, Equatable {
 
         let logLevel = try container.decodeIfPresent(LogLevel.self, forKey: .logLevel)
         self.logLevel = logLevel ?? .warn
-
-        self.payload = try Payload(from: decoder)
+        
+        do {
+            self.payload = try Payload(from: decoder)
+        } catch {
+            if lateInit != true {
+                throw error
+            }
+            
+            let emptyPayload = try JSONSerialization.data(withJSONObject: ["baseUrl":"", "clientId":""], options: [])
```

Comment:
what happens if TRY will fail

Reply:
Also why can't you create the empty payload directly? without jSONSerialization?

    self.payload = Payload(baseUrl: "", clientId: "")

Reply:
it's decoder constructor that have multiple options to be decoded.
What used here is overriding the current decoder for prevent falling if lateInit is true.

Reply:
If `lateInit` is `true`, we initialize the `baseUrl` and `clientId` at runtime for `Ionic Capacitor` without using `FronteggPlist` data, and we shouldn't throw an exception for missing `baseUrl` or `clientId`. 
If `lateInit` is false, we throw an exception at application launch due to invalid or missing configurations.